app/javascript/mastodon/utils/html.ts

```diff
@@ -10,64 +12,49 @@ export const unescapeHTML = (html: string) => {
   return wrapper.textContent;
 };
 
+interface AllowedTag {
+  /* True means allow, false disallows global attributes, string renames the attribute name for React. */
+  attributes?: Record<string, boolean | string>;
+  /* If false, the tag cannot have children. Undefined or true means allowed. */
+  children?: boolean;
+}
+
+type AllowedTagsType = {
+  [Tag in keyof React.ReactHTML]?: AllowedTag;
+};
+
+const GlobalAttributes: Record<string, boolean | string> = htmlConfig.global;
+const AllowedTags: AllowedTagsType = htmlConfig.tags;
```

Comment:
Should these actually be named in lower camelCase or all uppercase? I mistook them for a type at first glance